h2o-docs/src/product/data-science/algo-params/standardize.rst

```diff
@@ -1,7 +1,7 @@
 ``standardize``
 ---------------
 
-- Available in: Deep Learning, GLM, GAM, K-Means
+- Available in: Deep Learning, GLM, GAM, HGLM, K-Means
```

Comment:
Please add the following algos to the end of K-Means:

ANOVAGLM, ModelSelection